sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -311,20 +311,26 @@ public partial class KeyVaultSettingsClient
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Security.KeyVault.Administration.KeyVaultSetting>> UpdateSettingAsync(Azure.Security.KeyVault.Administration.KeyVaultSetting setting, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct SettingType : System.IEquatable<Azure.Security.KeyVault.Administration.SettingType>
+    public readonly partial struct KeyVaultSettingType : System.IEquatable<Azure.Security.KeyVault.Administration.KeyVaultSettingType>
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
-        public SettingType(string value) { throw null; }
-        public static Azure.Security.KeyVault.Administration.SettingType Boolean { get { throw null; } }
-        public bool Equals(Azure.Security.KeyVault.Administration.SettingType other) { throw null; }
+        public KeyVaultSettingType(string value) { throw null; }
+        public static Azure.Security.KeyVault.Administration.KeyVaultSettingType Boolean { get { throw null; } }
+        public bool Equals(Azure.Security.KeyVault.Administration.KeyVaultSettingType other) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override bool Equals(object obj) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override int GetHashCode() { throw null; }
-        public static bool operator ==(Azure.Security.KeyVault.Administration.SettingType left, Azure.Security.KeyVault.Administration.SettingType right) { throw null; }
-        public static implicit operator Azure.Security.KeyVault.Administration.SettingType (string value) { throw null; }
-        public static bool operator !=(Azure.Security.KeyVault.Administration.SettingType left, Azure.Security.KeyVault.Administration.SettingType right) { throw null; }
+        public static bool operator ==(Azure.Security.KeyVault.Administration.KeyVaultSettingType left, Azure.Security.KeyVault.Administration.KeyVaultSettingType right) { throw null; }
+        public static implicit operator Azure.Security.KeyVault.Administration.KeyVaultSettingType (string value) { throw null; }
+        public static bool operator !=(Azure.Security.KeyVault.Administration.KeyVaultSettingType left, Azure.Security.KeyVault.Administration.KeyVaultSettingType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class KeyVaultSettingValue
+    {
+        internal KeyVaultSettingValue() { }
```

Comment:
Should this be constructible?

Reply:
No, because 1) the `Value` property is read-only, and 2) it's set and controlled via constructing a `KeyVaultSetting` with only support types. Per previous discussion with @KrzysztofCwalina about this and conceptually matching what we did for the REST APIs, you can only create (post) a value that's supported, but we want to support any value coming back from the service since they could add new types in the future.